.env.sample

```diff
@@ -1,2 +1,3 @@
 PORT=8014
-DEBUG=False
\ No newline at end of file
+DEBUG=False
+ENA_ASSEMBLY_URL=https://www.ebi.ac.uk/ena/browser/view/
```

Comment:
This will have to be added to the https://gitlab.ebi.ac.uk/ensembl-web/newsite-api-config-setup/-/blob/main/prod/hl/metadata-api.config and in all the environments.